e2e/autoscript-suites/src/suites/authn-central-logout.test.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
+
+test('should login and logout with pingone', async ({ page }) => {
+  await page.goto('http://localhost:8443/');
+  await page.getByRole('link', { name: 'AuthN: Central Logout Ping' }).click();
+  const btn = page.getByRole('button', { name: 'Login' });
+  await btn.isVisible();
+  await btn.click({ clickCount: 1, delay: 300 });
```

Comment:
Isn't this quite fragile? Shouldn't we use a wait until element is visible or some non-time-based check? I'm a bit worried that pure time based "sleeps" are not reliable.

Reply:
we do wait for the button to be visible above, the delay was the only way for me to consistently click the button in the test. for some reason